new-log-viewer/src/components/Editor/MonacoInstance/actions.ts

```diff
@@ -0,0 +1,168 @@
+import * as monaco from "monaco-editor/esm/vs/editor/editor.api.js";
+
+import {Nullable} from "../../../typings/common";
+import {ActionType} from "../../../utils/actions";
+import {clamp} from "../../../utils/math";
+import type {
+    CursorExplicitPosChangeCallback,
+    CustomActionCallback,
+} from "./typings";
+
+
+const MIN_ZOOM_LEVEL = 1;
+const MAX_ZOOM_LEVEL = 10;
+const MOBILE_ZOOM_LEVEL_INCREMENT = 10;
+const MOBILE_ZOOM_LEVEL_DECREMENT = 1;
+const POSITION_CHANGE_DEBOUNCE_TIMEOUT_MILLIS = 50;
+
+/**
+ * Sets up an action that is triggered when the cursor position changes in a Monaco code editor.
+ *
+ * @param editor
+ * @param onCursorExplicitPosChange
+ */
+const setupCursorExplicitPosChangeAction = (
```

Comment:
How about `setupCursorExplicitPosChangeCallback` (or `Handler`)? The term "action" is a bit overused, so I think limiting it to things related to `ActionType` would be clearer.